code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -46,11 +35,7 @@
 	anchored = 1.0
 
 /obj/effect/gateway/Bumped(mob/M as mob|obj)
-	spawn(0)
-		return

```

Comment:
I thought this was here to override obj/Bumped() or effect/Bumped() or something. But apparently there's no non-empty parent Bumped(), so yeah let's remove.